src/EditorFeatures/CSharpTest/SymbolId/SymbolKeyTestBase.cs

```diff
@@ -195,13 +195,14 @@ internal static IList<ISymbol> GetSourceSymbols(CSharpCompilation compilation, b
         {
             var list = new List<ISymbol>();
             var localDumper = includeLocal ? new LocalSymbolDumper(compilation) : null;
-            GetSourceMemberSymbols(compilation.SourceModule.GlobalNamespace, list, localDumper);
+            GetSourceMemberSymbols(compilation.SourceModule.GlobalNamespace.GetPublicSymbol(), list, localDumper);
```

Comment:
> If we're not already aware of a bug tracking removal of the IVT, could you file one?

I think it would be best if you open that issue because you have a better idea what is it exactly you would want to be addressed, hence will be able to describe it much more clear. Etc.